scripts/tests/direct-download-roaming-network.sh

```diff
@@ -18,7 +18,7 @@ DOWNLOAD_PID=$!
 sleep 3 # Download a bit
 
 docker network disconnect firezone_app firezone-client-1 # Disconnect the client
-sleep 1
+sleep 3
```

Comment:
This one says "Disconnect the Client", so isn't the Client already connected for this one?

Reply:
Will remove that newline